### ISSUE TITLE
Project dashboard job pills contain no help text

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#5259]
The job pills shown in the project dashboard can be clicked to show the extended details of the job. However this is not communicated via a title or other help text, so users may be likely to miss it, or be confused on what clicking might do. Once the extended details are open, the panel itself should be read aloud. Focus has to stay on the pill to allow it to be closed again. Finally pills need to show focus so that it can be tracked throught the list.
